src/main/scala/mesosphere/marathon/core/launcher/impl/OfferProcessorImpl.scala

```diff
@@ -122,7 +122,7 @@ private[launcher] class OfferProcessorImpl(
   private[this] def saveTasks(ops: Seq[TaskOpWithSource], savingDeadline: Timestamp): Future[Seq[TaskOpWithSource]] = {
     def saveTask(taskOpWithSource: TaskOpWithSource): Future[Option[TaskOpWithSource]] = {
       val taskId = taskOpWithSource.taskId
-      log.info("Persisting TaskStateOp for [{}]", taskOpWithSource.taskId)
+      log.info("Processing {} for [{}]", Array[Object](taskOpWithSource.op.stateOp, taskOpWithSource.taskId): _*)
```

Comment:
Nit: we are using log with format and `s""` randomly. Seeing this call I would opt for `s""`